examples/rust-key-value/src/lib.rs

```diff
@@ -0,0 +1,46 @@
+use anyhow::Result;
+use http::{Method, StatusCode};
+use spin_sdk::{
+    http::{Request, Response},
+    http_component,
+    key_value::{Error, Store},
+};
+
+#[http_component]
+fn handle_request(req: Request) -> Result<Response> {
+    // Open the default key-value store
+    let store = Store::open("")?;
```

Comment:
Is there any way we can hide the special empty string from consumers?  `Store::default()` and `Store::named/Store::open` maybe, though it might be a bit profligate?

Reply:
Yes, that was my suggestion when we discussed it.  IIRC, @lann wanted to use the empty string so that runtime config files could refer to it explicitly, but perhaps we could just have special runtime config file syntax for configuring the default store?

Reply:
I'm fine with `Store::default()` being equivalent to `Store::open("")`. I don't recall being against it but if I was then I'm disagreeing with previous-me.

Reply:
Oh to be clear I'm fine with it _in the SDK_. I think it should be explicit at the Wasm interface level.